cmd/kubeadm/app/master/manifests.go

```diff
@@ -305,7 +305,7 @@ func getAPIServerCommand(cfg *kubeadmapi.MasterConfiguration, selfHosted bool) [
 
 	command = append(getComponentBaseCommand(apiServer),
 		"--insecure-bind-address=127.0.0.1",
-		"--admission-control=NamespaceLifecycle,LimitRanger,ServiceAccount,PersistentVolumeLabel,DefaultStorageClass,ResourceQuota",
+		"--admission-control="+cfg.AdmissionControl,
```

Comment:
could you do this instead here:
```go
for k, v := range cfg.ExtraAPIServerArgs {
    command = append(command, fmt.Sprintf("--%s=%s", k, v))
}
// These arguments are defaults that can be overridden by ExtraAPIServerArgs
configurableDefaultCommands := map[string]string{
    "admission-control": kubeadmconstants.DefaultAdmissionControl,
    "insecure-bind-address": "127.0.0.1",
    "allow-privileged": "true",
}
for k, v := range configurableDefaultCommands {
    if _, overrideExists := cfg.ExtraAPIServerArgs[k]; !overrideExists {
        command = append(command, fmt.Sprintf("--%s=%s", k, v))
    }
}
```

the logic I outlined here should probably be a separate function that takes `cfg.ExtraAPIServerArgs` and `configurableDefaultCommands` as inputs and outputs a `[]string`